lib/crypt_keeper/provider/postgres_pgp.rb

```diff
@@ -42,8 +42,14 @@ def decrypt(value)
       end
 
       def search(records, field, criteria)
-        records.where("(pgp_sym_decrypt(cast(\"#{field}\" AS bytea), ?) = ?)",
-          key, criteria)
+        if criteria.present?
+          records
+            .where.not("TRIM(BOTH FROM #{field}) = ?", "")
+            .where("(pgp_sym_decrypt(cast(\"#{field}\" AS bytea), ?) = ?)",
+                   key, criteria)
+        else
+          records.where("#{field}" => criteria)
```

Comment:
Would this work just as well by just using the `field` variable, instead of interpolating it into a string? I can see that if the value of `field` is `nil` or `""`, this will prevent code from raising an error and return an AR relation, but I'm not sure if this behavior is desirable as it seems to subsequently raise an error when any non-AR method is called on the relation returned from the query. Please take this question in stride, as I've never worked with this gem, but if `field` is `nil` or `""`, this seems like a reasonable place for a loud failure.